learningresources/search_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+from haystack.indexes \
+    import SearchIndex, Indexable, CharField  # pragma: no cover
```

Comment:
I agree with you on these things. I did them this way because:

* The coverage score is zero. As a config file it doesn't get covered at all.
* Writing the imports as "one line" was done because the `pragma` tag has no scope larger than one line.

I'll fix the imports and put the `pragma` on the first or last line and see if coverage respects it.